src/cli/commands/test/formatters/legacy-format-issue.ts

```diff
@@ -76,7 +77,13 @@ export function formatIssues(
   );
 }
 
-function createSeverityBasedIssueHeading(severity, type, packageName, isNew) {
+function createSeverityBasedIssueHeading(
```

Comment:
did you make sure that you updated all the places that use this function?

Reply:
It's a local function to that file with one place, though there is another with that name that is used for `snyk wizard`. I wonder if we need to change it too though... 🤔

Reply:
Yeah, the function isn't exported anywhere. 

nit: However, I think originally 4 arguments was a lot already, and I would make it an object.

```suggestion
function createSeverityBasedIssueHeading({ severity, originalSeverity, type, packageName, isNew })
```